swagger.go

```diff
@@ -61,6 +62,16 @@ func CustomWrapHandler(config *Config, h *webdav.Handler) gin.HandlerFunc {
 		prefix := matches[1]
 		h.Prefix = prefix
 
+		if strings.HasSuffix(path, ".html") {
+			c.Header("Content-Type", "text/html; charset=utf-8")
+		} else if strings.HasSuffix(path, ".css") {
+			c.Header("Content-Type", "text/css; charset=utf-8")
+		} else if strings.HasSuffix(path, ".js") {
+			c.Header("Content-Type", "application/javascript")
+		} else if strings.HasSuffix(path, ".json") {
+			c.Header("Content-Type", "application/json")
+		}
```

Comment:
This part seems like can have better solutions.  It can be improved for the new PR.

Reply:
Yes, as I said it's not the best solution, but it works. I don't know why header was overwritten with `application/x-gzip` value before. This only fixes consequences, not origins.